js/libs/form.js

```diff
@@ -241,13 +242,33 @@ Fliplet.Widget.instance('form-builder', function(data) {
         return found;
       },
       onSubmit: function() {
+        var $vm = this;
+        var formData = {};
+
         Fliplet.Analytics.trackEvent({
           category: 'form',
           action: 'submit'
         });
 
-        var $vm = this;
-        var formData = {};
+        // form validation
+        $vm.isFormValid = true;
+
+        $vm.$children.forEach(function (inputField) {
+
+          // check is not buttons
```

Comment:
what does this comment mean? please rewrite

Reply:
fixed